packages/compiler/src/myst.ts

```diff
@@ -0,0 +1,41 @@
+import { micromark } from "micromark";
+
+export function parsePanels (contents) {
```

Comment:
It would be great to add one or more test cases for this function in `packages/compiler/__tests__`

Reply:
Added ```myst.tests.js``` to ```packages/compiler/__tests__```